editions/tw5.com/tiddlers/howtos/How to export tiddlers.tid

```diff
@@ -0,0 +1,30 @@
+created: 20151006145934884
+modified: 20151006155211988
+tags: Learning
```

Comment:
I was thinking of it as a "how-to", and this is where all the other how-tos are located. But elevating it to the more prominent  "Working with TiddlyWiki" category would be fine by me. "Exporting Tiddlers" would be a good title -- I think it's time to lose the camel-case whenever possible. I note that none of the existing "Working with..." tiddlers are in camel-case.